src/current/cockroachcloud/export-logs.md

```diff
@@ -342,7 +345,7 @@ Perform the following steps to enable log export from your CockroachDB {{ site.d
             This configuration:
             - Enables [redaction]({% link {{site.current_cloud_version}}/configure-logs.md %}#redact-logs) globally for all log entries emitted to GCP Cloud Logging.
             - Does not send log entries in the `SESSIONS` and `SQL_PERF` logging channels.
-            - Sends log entries in the `SQL_SCHEMA` and `SQL_EXEC` [logging channels]({% link {{site.current_cloud_version}}/logging-overview.md %}#logging-channels)) to a GCP Cloud Logging log group named `sql`, and overrides (disables) the global redaction configuration for just these two log channels only.
+            - Sends log entries in the `SQL_SCHEMA` and `SQL_EXEC` [logging channels]({% link {{site.current_cloud_version}}/logging-overview.md %}#logging-channels) to a GCP Clo ud Logging log group named `sql`, and overrides (disables) the global redaction configuration for just these two log channels only.
```

Comment:
This is introducing a typo: `Clo ud`

AFAICT this typo is not in `main` nor in #18875 

?

(PS maybe i'm missing something here, I haven't worked in this branch in a long time)